packages/vscode-extension/test/suite/envChecker/cases/all.ts

```diff
@@ -124,33 +154,62 @@ suite("All checkers E2E test", async () => {
       chai.assert.isTrue(
         await dotnetUtils.hasAnyDotnetVersions(dotnetExecPath!, dotnetUtils.dotnetSupportedVersions)
       );
+
+      chai.assert.isFalse(await isNonEmptyDir(backendOutputPath));
+      await backendExtensionsInstaller.install(
+        backendProjectDir,
+        testCsprojFileName,
+        testOutputDirName
+      );
+      chai.assert.isTrue(await isNonEmptyDir(backendOutputPath));
     }
   });
 
   test("All disabled", async function(this: Mocha.Context) {
-    const nodeVersion = await nodeUtils.getNodeVersion();
-    if (nodeVersion != null) {
-      this.skip();
-    }
+    const [checker, _, dotnetChecker, backendExtensionsInstaller] = createTestChecker(
+      true,
+      false,
+      false,
+      false,
+      false
+    );
+    const shouldContinue = await checker.resolve();
+    const dotnetExecPath = await dotnetChecker.getDotnetExecPath();
+
+    chai.assert.isTrue(shouldContinue);
+    chai.assert.isNotNull(dotnetExecPath);
+    chai.assert.equal(dotnetExecPath!, dotnetUtils.dotnetCommand);
+
+    chai.assert.isFalse(await isNonEmptyDir(backendOutputPath));
+
     if (
       await dotnetUtils.hasAnyDotnetVersions(
         dotnetUtils.dotnetCommand,
         dotnetUtils.dotnetSupportedVersions
       )
     ) {
-      this.skip();
+      await backendExtensionsInstaller.install(
+        backendProjectDir,
+        testCsprojFileName,
+        testOutputDirName
+      );
+      chai.assert.isTrue(await isNonEmptyDir(backendOutputPath));
+    } else {
+      // If dotnet command is not found, spawn will throw an ENOENT error
```

Comment:
If dotnet command can't found, it means above operation already fails and the test case should failed immediately instead of running below operation?